source/stylesheets/refills/unstyled/_hover-menu.scss

```diff
@@ -0,0 +1,60 @@
+.hover-menu {
+  $large-screen: em(860) !default;
+  $submenu-width: 12em;
+
+  overflow: visible;
+  padding-right: 0;
+
+  @include media($large-screen) {
+    position: relative;
+  }
+
+  .hover-menu-item {
+    left: 0;
+    position: relative;
+
+    &::after {
+      content: "›";
+      padding-left: 0.5em;
+    }
+  }
+
+  .submenu {
+    display: none;
+    margin: 0;
+    padding-left: 0;
+
+    @include media($large-screen) {
+      left: 0;
+      position: absolute;
+    }
+  }
+
+  .submenu li {
```

Comment:
Merge rule `.submenu li` with rule on line 22<br>Selector should have depth of applicability no greater than 2, but was 3